clang/tools/clang-format/ClangFormat.cpp

```diff
@@ -284,27 +284,23 @@ static bool fillRanges(MemoryBuffer *Code,
     errs() << "error: number of -offset and -length arguments must match.\n";
```

Comment:
Is it maybe worth handling the `Offsets.empty() && Lengths.empty()` case by its own, and then dropping the `if (I < Size)`?

Reply:
Actually, we can achieve that by handling `Offsets.size() == 1 && Lengths.empty()` separately.